docs/conversion_examples_gallery/sorting/plexon.rst

```diff
@@ -0,0 +1,31 @@
+Plexon sorting data conversion
+---------------------------
+
+Install NeuroConv with the additional dependencies necessary for reading Plexon data.
+
+.. code-block:: bash
+
+    pip install neuroconv[plexon]
+
+Convert Plexon spiking data (.plx) to NWB using :py:class:`~.neuroconv.datainterfaces.ecephys.plexon.plexondatainterface.PlexonSortingInterface`.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from neuroconv.datainterfaces import PlexonSortingInterface
+    >>>
+    >>> file_path = f"{ECEPHY_DATA_PATH}/plexon/File_plexon_2.plx"
+    >>> # Change the file_path to the location in your system
+    >>> interface = PlexonSortingInterface(file_path=file_path, verbose=False)
+    >>>
+    >>> # Extract what metadata we can from the source files
+    >>> metadata = interface.get_metadata()
+    >>> # For data provenance we add the time zone information to the conversion
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
```

Comment:
If we can indeed get the datetime from the metadata, let's illustrate using it

Reply:
I'm a tad concerned that since it's an 'offline sorter' the datetime might just be whenever the method was run, not necessarily when it was recorded...

waiting to hear back on how its used in the specific experiment

Reply:
Szonja, can you change this to use the date time extracted from the metadata and add a timezone?

Reply:
For the tutorial, not the Tye lab conversion